templates/index.html

```diff
@@ -1,98 +1,272 @@
-				data: {
-					imageBase64: dataURL
-				},
-				success: function (result) {
-					console.log(result);
-				}
-			});
-			e.preventDefault();
-		}, false);
-
-		clearPhoto();
-
-		// create image then convert it into format usable by <img>
-		function clearPhoto() {
-			context.fillStyle = "#666";
-			context.fillRect(0, 0, canvas.width, canvas.height);
-
-			const data = canvas.toDataURL("image/png");
-			photo.setAttribute("src", data);
-		}
-
-		// capture frame from stream
-		function capture() {
-			if (width && height) {
-				canvas.width = width;
-				canvas.height = height;
-				context.drawImage(video, 0, 0, width, height);
-
-				const data = canvas.toDataURL("image/png");
-				photo.setAttribute("src", data);
-			} else {
-				clearPhoto();
-			}
-		}
-	</script>
+    <div class="sidebar">
+        <div class="sidebar-container">
+         <div class="brand">
+             <h3>
+                 <!-- <span class="fa-brands fa-staylinked"></span> -->
+								 <!-- insert HCP logo here -->
+                 HAYR
+             </h3>
+             <i class="fas fa-times"></i>
+         </div>
+
+         <div class="sidebar-avatar">
+             <div>
+                 <!-- <img  src="youtubelogo.jpg" alt="alt"> -->
+             </div>
+             <div class="avatar-info">
+                 <div class="avatar-text">
+                     <h4>HAYR</h4>
+                     <small></small>
+                 </div>
+
+                <i class="fa-solid fa-angle-double-down"></i>
+             </div>
+         </div>
+
+         <div class="sidebar-menu">
+                <ul>
+                    <li><a  href="#">
+                        <i class="fa-solid fa-dashboard"></i>
+                        <span>Dashboard</span>
+                    </a> </li>
+                    <li><a href="#">
+                        <i class="fa-solid fa-video-camera"></i>
+                        <span>Videos</span>
+                    </a></li>
+                    <li><a class="active"  href="#">
+                        <i class="fa-solid fa-chart-bar"></i>
+                        <span>Analytics</span>
+                    </a></li>
+                    <li><a href="#">
+                        <i class="fa-solid fa-calendar"></i>
+                        <span>Schedule</span>
+                    </a></li>
+                    <li><a href="#">
+                        <i class="fa-solid fa-user"></i>
+                        <span>Account</span>
+                    </a></li>
+                </ul>
+         </div>
+
+         <!-- <div class="sidebar-card">
+
+              <div class="side-card-item" >
+                 <h4>Make Adsense</h4>
+                 <p>Add ads to your videos to earn money</p>
+             </div>
+              <button class="btn btn-main btn-block">
+                  Create Now
+              </button>
+
+         </div> -->
+
+        </div>
+    </div>
+
+
+    <div class="main-content">
+
```

Comment:
Please remove unnecessary new lines (unless they are intended; eg. to separate different parts/components)